Source/BoundaryConditions/WarpXEvolvePML.cpp

```diff
@@ -299,29 +302,24 @@ WarpX::DampJPML (int lev, PatchType patch_type)
             int const zs_lo = sigba[mfi].sigma_star_cumsum_fac[1].lo();
 #endif
 
+            bool const eb_enabled = m_eb_enabled;
             amrex::ParallelFor( tjx, tjy, tjz,
                 [=] AMREX_GPU_DEVICE (int i, int j, int k) {
-#ifdef AMREX_USE_EB
-                    if(pml_lxfab(i, j, k) <= 0) return;
-#endif
+                    if (eb_enabled && pml_lxfab(i, j, k) <= 0) return;
```

Comment:
Alternative that is more compact:
```suggestion
                    if (pml_lxfab && pml_lxfab(i, j, k) <= 0) return;
```

Reply:
done